setup.py

```diff
@@ -0,0 +1,16 @@
+from pathlib import Path
+from setuptools import setup, find_packages
+
+setup(
+    name="medicine",
+    version="0.0.1",
```

Comment:
pierwsze co znalazłem na google: https://www.freecodecamp.org/news/semantic-versioning-1fd6f57749f7/
Nie jest to najlepszy opis, ale przedstawia to co istotne